index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=\, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="parent">
+        <div class="part">
+            <img src="https://images.unsplash.com/photo-1587088132102-813d79451b64?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=934&q=80"
+                alt="Houston">
+            <h2>Houston </h2>
+            <p>
+                T6|CONDUIT|INFRASTRUCRE
+            </p>
+            <div class="t">
+                <p>
+                    T
+                </p>
+            </div>
```

Comment:
instead of adding 'T' in a separate div, you can create a div which contains the image and 'T'. 
So you can put the position for 'T' absolute according to this div .

ex. 
` <div class="image-container">
          <img
            src="https://images.unsplash.com/photo-1587088132102-813d79451b64?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=934&q=80"
            alt="Houston"
          />
          <div class="t">
            <p>T</p>
          </div>
        </div>`

and make sure  the position of `image-container` is `relative`

Reply:
done